tests/unit_tests/corsika/test_corsika_output.py

```diff
@@ -717,3 +706,152 @@ def test_event_2D_histogram(corsika_output_instance_set_histograms):
+                      "hist_1D_photon_time_distr_all_tels.ecsv",
+                      "hist_1D_photon_time_distr_all_tels.ecsv",
+                      "hist_1D_photon_per_event_distr_all_tels.ecsv",
+                      "hist_1D_photon_per_telescope_distr_all_tels.ecsv"
+                      ]:
+        assert io_handler.get_output_directory(test=True).joinpath(file_name).exists()
+
+
+def test_export_2D_histograms(corsika_output_instance_set_histograms, io_handler):
+    corsika_output_instance_set_histograms._export_2D_histograms(
+        output_dir=io_handler.get_output_directory(test=True))
+
+    for file_name in ["hist_2D_photon_direction_distr_all_tels.ecsv",
+                      "hist_2D_photon_time_altitude_distr_all_tels.ecsv",
+                      "hist_2D_photon_telescope_event_distr_all_tels.ecsv",
+                      "hist_2D_photon_count_distr_all_tels.ecsv",
+                      "hist_2D_photon_density_distr_all_tels.ecsv",
+                      ]:
+        assert io_handler.get_output_directory(test=True).joinpath(file_name).exists()
+
+
+def test_dict_2D_distributions(corsika_output_instance_set_histograms):
+    expected_dict_2D_distributions = {
+        "counts": {
+            "function": "get_2D_photon_direction_distr",
+            "file name": "hist_2D_photon_direction_distr",
+            "title": "Incoming directive cosines",
+            "x edges": "x directive cosinus",
+            "x edges unit": u.dimensionless_unscaled,
+            "y edges": "y directive cosinus",
+            "y edges unit": u.dimensionless_unscaled,
+        }
+    }
+    assert corsika_output_instance_set_histograms._dict_2D_distributions["counts"] == \
+           expected_dict_2D_distributions["counts"]
+
+
+def test_fill_ecsv_table_1D(corsika_output_instance_set_histograms):
+    hist = np.array([1, 2, 3])
+    x_edges = np.array([1, 2, 3, 4])
+    y_edges = None
+    x_label = 'test_x_label'
+    y_label = None
+
+    table = corsika_output_instance_set_histograms.fill_ecsv_table(hist, x_edges, y_edges, x_label,
+                                                                   y_label)
+
+    assert all(table[x_label] == x_edges[:-1])
+    assert all(table['Values'] == hist)
+
+
+def test_fill_ecsv_table_2D(corsika_output_instance_set_histograms):
+    hist = np.array([[1, 2], [3, 4]])
+    x_edges = np.array([1, 2, 3])
+    y_edges = np.array([1, 2, 3])
+    x_label = 'test_x_label'
+    y_label = 'test_y_label'
+
+    table = corsika_output_instance_set_histograms.fill_ecsv_table(hist, x_edges, y_edges, x_label,
+                                                                   y_label)
+
+    assert all(table[x_label] == np.array([1, 2, 1, 2]))
+    assert all(table[y_label] == np.array([1, 1, 2, 2]))
+    assert all(table['Values'] == hist.flatten())
+
+
+def test_export_event_header_1D_histogram(corsika_output_instance_set_histograms, io_handler):
+    CORSIKA_event_header_example = {
```

Comment:
Because you wrote CORSIKA with capital letters, GitHub decides to colour it as if it is a class name. Could you change it to `corsika` instead? It would also be consistent with the rest of the code.